frontend/src/components/Navigation/index.jsx

```diff
@@ -1,75 +1,119 @@
-import React, {Component} from 'react';
-import logo from '../../static/images/logo/logo.svg';
-import {Link} from 'react-router-dom';
-import {userActions} from '../../actions/user.actions';
-import {connect} from 'react-redux';
+import React, { Component } from "react";
```

Comment:
why did you change from ' to ",
it dosne't follow the eslint rules.
this changes are not expected. same go for spaces